spec/ddtrace/contrib/elasticsearch/patcher_spec.rb

```diff
@@ -18,6 +20,13 @@
       c.use :elasticsearch, configuration_options
     end
 
+    # LibCurl native thread
+    allow(::Ethon::Easy).to receive(:new).and_wrap_original do |method, *args, &block|
```

Comment:
This one is interesting: ElasticSearch detects any available HTTP client installed, and finds the ones with best performance: https://github.com/elastic/elasticsearch-ruby/tree/master/elasticsearch-transport#transport-implementations
One of the possible clients is `Typhoeus`, which is built on top of `Ethon`.

So when running this test with or without `Typhoeus`, the results will be internally different.

Regardless, I passed an option to ElasticSearch so that it always uses Net::HTTP in our tests.